api/src/paths/project/{projectId}/survey/{surveyId}/update.ts

```diff
@@ -167,6 +177,25 @@ PUT.apiDoc = {
                 }
               }
             },
+            partnerships: {
+              title: 'Survey partnerships',
+              type: 'object',
```

Comment:
Are the internal properties required (indigenous_partnerships, stakeholder_partnerships)?
Could add a required section at this level as well, if they are.

Reply:
Same for the view and create endpoints*

Reply:
Updated these files to xplicitly declared which fields were required. 👍